### PR TITLE
[FEAT] Sound Effects

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

This PR adds the following sounds effects to the game:
Total size for the SFX folder including all the sounds is 300Kb.

* Chopping/falling tree
* Bank
* Wishing Well
* Blacksmith
* Planting/harvesting
* Mining (stone, iron, gold)
* Team Donation
* Shop
* Kitchen
* Frog 

Also added in this PR is Harvesting, the new song for V2.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Manually tested

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by AncientHorse** [comment]
This is so awesome.

**Comment by adamhannigan** [comment]
@Rofrtd Awesome work mate 👏 

2 small suggestions:

1. When opening the Blacksmith, only play the sound effect once instead of 3 times
2. The tree falling is a bit overwhelming, can we soften it and make it shorter?

File: src/features/bakery/Bakery.tsx [hunk 1]
```diff
@@ -20,6 +21,13 @@ export const Bakery: React.FC = () => {
 
   const isNotReadOnly = !gameState.matches("readonly");
 
+  const kitchenAudio = new Audio(kitchenMP3);
```
Comment (adamhannigan) [inline src/features/bakery/Bakery.tsx:24]: Do we need to initialise this every time the React component rerenders?

Can we move this inside of the `open` function on line 27?

File: src/assets/songs/playlist.ts [hunk 2]
```diff
@@ -1,6 +1,12 @@
 import willow_tree from "./willow_tree.mp3";
+import harvesting from "./harvesting.mp3";
 // Playlist hardcoded, but eventually it will be fetched from the server
 const song_list = [
+  {
+    artist: "Romy & Rick",
+    name: "Harvesting",
+    path: harvesting,
```
Comment (adamhannigan) [inline src/assets/songs/playlist.ts:8]: That is a tune ❤️ 💪

**Review by adamhannigan — APPROVED** [review]
Love it mate! Music is sick.

Not needed to address now, but is there a way we can reduce the size of the new 1.86MB song?

**Pull request merged**